### ISSUE TITLE
Implications of artifact evaluation on paper acceptance

### ISSUE THREAD

**Issue by sbaltes** [acmsigsoft/artifact-evaluation#9]
What happens when reviewers find major flaws during artifact evaluation?
Can artifact evaluation impact paper acceptance?

**Comment by gousiosg** [comment]
My opinion is that it must go without saying that if the artifact evaluation reveals flaws that the authors cannot explain, then the paper acceptance must be reexamined. 

Ideally, the artefact evaluation should be part of the paper review process and done in parallel. Evaluators should be able to give reviewers their opinion while papers are being discussed.

**Comment by sbaltes** [comment]
I totally agree, but I remember that this question was controversially discussed at a previous ICSE town hall meeting (I think it was in Buenos Aires).

**Comment by jon-bell** [comment]
One major concern here: does this create a disincentive for authors to submit artifacts? That is, given the choice between:
1. Publish paper without artifact, get paper accepted and 
2. Take time to prepare artifact, risk paper gets rejected based on artifact
Why would an author take choice 1?

**Comment by sbaltes** [comment]
I think the long term goal should be that 1. is not possible without justification (e.g., NDA with company). So the question is what to do in the meantime...

**Comment by dgraziotin** [comment]
@sbaltes this is something we aim to push for in the SIGSOFT open science policies (will put them out here as well)
